envi/tests/test_arch_arm.py

```diff
@@ -271,52 +800,90 @@ def test_envi_arm_operands(self):
 
         
     def test_envi_arm_assorted_instrs(self):
-
-        #archmod = envi.getArchModule("h8")
+        #setup initial work space for test
         vw = vivisect.VivWorkspace()
         vw.setMeta("Architecture", "arm")
         vw.addMemoryMap(0, 7, 'firmware', '\xff' * 16384*1024)
         vw.addMemoryMap(0x400000, 7, 'firmware', '\xff' * 16384*1024)
         emu = vw.getEmulator()
-        emu.logread = emu.logwrite = True
-
+        #took out next line while testing routine
+        #mu.logread = 
```

Comment:
Was pulled before I was doing the unit test and was running the code directly. I will put it back. 
Not sure where the e went... 

Date: Tue, 7 Jun 2016 10:55:28 -0700
From: notifications@github.com
To: vivisect@noreply.github.com
CC: redheadedrod@hotmail.com; author@noreply.github.com
Subject: Re: [atlas0fd00m/vivisect] Unittest (#5)

In envi/tests/test_arch_arm.py:

>          vw = vivisect.VivWorkspace()
>          vw.setMeta("Architecture", "arm")
>          vw.addMemoryMap(0, 7, 'firmware', '\xff' * 16384*1024)
>          vw.addMemoryMap(0x400000, 7, 'firmware', '\xff' * 16384*1024)
>          emu = vw.getEmulator()
> -        emu.logread = emu.logwrite = True
> -
> +        #took out next line while testing routine
> +        #mu.logread = 

this was actually "emu.logread", not "mu.logread"

why was it taken out?


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.